Packs/EmailCommunication/Scripts/DisplayEmailHtmlThread/DisplayEmailHtmlThread.py

```diff
@@ -0,0 +1,130 @@
+        email_subject: The email subject.
+        html_body: The email HTML body.
+        email_time: The time the email was received
+        attachments: List of attachment entries
+        attachment_names: String of attachment names
+    Returns:
+        str. Email reply.
+    """
+
+    single_reply = f'<html><body><b>From:</b> {email_from}<br><b>To:</b> {email_to}<br><b>CC:</b> {email_cc}<br>' \
+                   f'<b>Subject:</b> {email_subject}<br><b>Email Time:</b> {email_time}<br>' \
+                   f'<b>Attachments:</b> {attachment_names}</body></html>'
+    if attachments:
+        attachment_names = [attachment.get('name', '') for attachment in attachments]
+        single_reply += f'Attachments: {attachment_names}\n'
+
+    single_reply += f'\n{html_body}\n<hr style="width:98%;text-align:center;height:3px;border-width:0;' \
+                    f'background-color:#cccccc">\n\n'
+
+    return single_reply
+
+
+def html_cleanup(full_thread_html):
+    """
+        Moves various HTML tags so the final output is a single HTML document
+    Args:
+        full_thread_html (string): The concatenated string of all email HTML
+    Returns:
+        str. Full email thread HTML
+    """
+    # Remove HTML tags within the string that should only occur once
+    full_thread_html = re.sub('(?i)<!DOCTYPE html>', '', full_thread_html)
+    full_thread_html = re.sub('(?i)<.?html>', '', full_thread_html)
+    full_thread_html = re.sub('(?i)<.?body>', '', full_thread_html)
+
+    # Place needed HTML tags in their appropriate locations
+    final_html_result = f'<!DOCTYPE html>\n<html>\n<body>\n{full_thread_html}\n</body>\n</html>'
+
+    return final_html_result
+
+
+def main():
+    incident = demisto.incident()
+    custom_fields = incident.get('CustomFields')
+    thread_number = custom_fields.get('emailselectedthread')
+    incident_context = demisto.context()
+    email_threads = incident_context.get('EmailThreads', {})
+
+    if len(email_threads) == 0:
+        return_error('This incident does not contain any email threads yet.')
+
+    if isinstance(email_threads, dict):
+        email_threads = [email_threads]
+
+    thread_exists = False
+    thread_items = []
+    full_thread_html = str()
+
+    for thread in email_threads:
+        if str(thread['EmailCommsThreadNumber']) == str(thread_number):
+
+            thread_exists = True
+            thread_dict = {
+                'email_from': thread.get('EmailFrom', None),
+                'email_cc': thread.get('EmailCC', None),
+                'email_to': thread.get('EmailTo', None),
+                'email_subject': thread.get('EmailSubject', None),
+                'email_html': thread.get('EmailHTML', None),
+                'email_time': thread.get('MessageTime', None),
+                'email_attachments': thread.get('EmailAttachments', None),
+            }
+            thread_items.append(thread_dict)
+
+    attachments = None  # incident.get('attachment', {})
```

Comment:
This was a remnant from earlier work when attachment data was being stored on on a per-Incident level instead of a per-message level.  I have removed the unneeded references to this attachment field.